lib/cronitor/monitor.rb

```diff
@@ -164,11 +157,11 @@ def monitor_api_url
     private
 
     def fetch
-      if !api_key
-        raise Cronitor::Error("No API key detected. Set Cronitor.api_key or initialize Monitor with the api_key kwarg")
-      end
+      return Cronitor.logger.error(
+        "No API key detected. Set Cronitor.api_key or initialize Monitor with the api_key kwarg"
+      ) if !api_key
```

Comment:
So I appreciate this, but for myself (and per RuboCop I think) if it’s a multi-line declaration, use the multi-line style:

```ruby
if $thing
  do stuff
end
```